src/rmarkdown.ts

```diff
@@ -1,23 +1,33 @@
 import {
   CancellationToken, CodeLens, CodeLensProvider,
   CompletionItem, CompletionItemProvider,
-  Event, EventEmitter, Position, Range, TextDocument, TextEditorDecorationType, window, Selection
+  Event, EventEmitter, Position, Range, TextDocument, TextEditorDecorationType, window, Selection, languages
 } from 'vscode';
 import { runChunksInTerm } from './rTerminal';
 import { config } from './util';
 
-function isChunkStartLine(text: string) {
-  if (text.match(/^\s*```+\s*\{\w+\s*.*$/g)) {
-    return true;
+function isRDocument(document: TextDocument) {
+  return (languages.match('r', document) > 0);
```

Comment:
Is there a difference using `document.languageId` instead of `languages.match('r', document) > 0`>

Reply:
I haven't found any difference, thanks for picking that up! Not sure what the use case would be if languageId exists